pages/_app.jsx

```diff
@@ -28,11 +31,17 @@ const customBreakpoints = {
   lg: 1024,
   xl: 1440,
 };
+const colors = {
```

Comment:
why did you add this object here? and why it includes only one color?

Reply:
I was looking to see if the colour change would work, just testing, I've already fixed it